src/python/turicreate/toolkits/sound_classifier/_tf_sound_classifier.py

```diff
@@ -0,0 +1,233 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2019 Apple Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can
+# be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+from __future__ import print_function as _
+from __future__ import division as _
+from __future__ import absolute_import as _
+from turicreate.util import _ProgressTablePrinter
+import tensorflow as _tf
+import numpy as _np
+import time as _time
+from .._tf_model import TensorFlowModel
+import turicreate.toolkits._tf_utils as _utils
+
+import tensorflow.compat.v1 as _tf
+_tf.disable_v2_behavior()
+
+
+class SoundClassifierTensorFlowModel(TensorFlowModel):
```

Comment:
If SoundClassifierTensorFlowModel inherits from TensorFlowModel, you will also have to implement `set_learning_rate()`

Reply:
Yeah sorry I thought it was high level stuff.

Reply:
There are still some pieces to be done: learning rate, data iterator etc

Reply:
There is no strict need to implement `set_learning_rate` now anyway. We're not going to call this from C++ in the immediate future